### ISSUE TITLE
Navidrome support 

### ISSUE THREAD

**Issue by idhirandar** [icefields/Power-Ampache-2#168]
Is there any possibility to add navidrome backend in future

**Comment by icefields** [comment]
Yes. Navidrome uses the Subsonic API, I will start working on that after I'm done with the Chromecast / Android-Auto plugin for Power Ampache 2.

**Comment by candideu** [comment]
This would be lovely!
